package.json

```diff
@@ -14,7 +14,9 @@
     "build/src",
     "AUTHORS",
     "CONTRIBUTORS",
-    "LICENSE"
+    "LICENSE",
```

Comment:
Both `LICENSE` and `README` are included automatically during `npm publish`, but it doesn't hurt anything either to have them here. Just mentioning it.